spire/templates/jump-server.yml

```diff
@@ -158,8 +156,6 @@ Resources:
                 export PRX_SHARED_POSTGRES_PORT=${SharedAuroraPostgresqlPort}
                 export PRX_SHARED_APP_REDIS_HOST=${SharedAppRedisEndpointAddress}
                 export PRX_SHARED_APP_REDIS_PORT=${SharedAppRedisEndpointPort}
-                export PRX_SHARED_REDIS_HOST=${SharedRedisReplicationGroupEndpointAddress}
-                export PRX_SHARED_REDIS_PORT=${SharedRedisReplicationGroupEndpointPort}
                 export PRX_DOVETAIL_REDIS_HOST=${DovetailRedisReplicationGroupEndpointAddress}
                 export PRX_DOVETAIL_REDIS_PORT=${DovetailRedisReplicationGroupEndpointPort}
```

Comment:
This will require an awstunnel update.  But, I think people hardly ever tunnel to get to redis anyways.